velox/functions/sparksql/aggregates/tests/BloomFilterAggAggregateTest.cpp

```diff
@@ -86,4 +86,19 @@ TEST_F(BloomFilterAggAggregateTest, nullBloomFilter) {
           vectors, {}, {"bloom_filter_agg(c0, 5, 64)"}, expectedFake),
       "First argument of bloom_filter_agg cannot be null");
 }
+
+TEST_F(BloomFilterAggAggregateTest, config) {
+  auto vector = {makeRowVector({makeFlatVector<int64_t>(
+      100, [](vector_size_t row) { return row % 9; })})};
+  auto bloomFilter = getSerializedBloomFilter(100);
+  auto expected = {
+      makeRowVector({makeConstant<StringView>(StringView(bloomFilter), 1)})};
+
+  testAggregations(
```

Comment:
How do we ensure that config values are being honored?

Reply:
This config will decide the bloom filter capacity, the expected value is the serialized bloom filter, it should be consistent.

Reply:
Add the test, but looks not clear. Because `AssertQueryBuilder#assertResults` won't throw exception in `testAggregations` to check.

Reply:
> Because AssertQueryBuilder#assertResults won't throw exception in testAggregations to check.

Is this a bug in AssertQueryBuilder#assertResults?

Reply:
No, because it uses ADD_FAILURE to fail directly. https://github.com/facebookincubator/velox/blob/main/velox/exec/tests/utils/QueryAssertions.cpp#L978

Reply:
```
../../velox/exec/tests/utils/QueryAssertions.cpp:1013: Failure
Failed
Expected 1, got 1
1 extra rows, 1 missing rows
1 of extra rows:

```